docs/HowToUpdate.md

```diff
@@ -0,0 +1,65 @@
+# How to Update
+
+This document outlines how to update parts of the website. See `Contributing.md` to see how to contribute to the repo. 
+
+## Exec/staff list
+
+Open `src/pages/about/components/execList.js`. This file contains the full list of exec and staff including their names,
+roles, and description. By editing this file you can update the contents of the site.
+
+### Updating
+
+Edit the content in `execList.js`.
+
+To update profile pictures, replace the corresponding images in `src/pages/about/assets`. Make sure the file names match
+the imported images in `execList.js`.
+
+### Adding
+
+Add a new individual in `execList.js`. Be sure to include the position, name, image and description fields.
+
+To upload an image, add the image to `src/pages/about/assets`. Then import the image in `execList.js`:
+
+```javascript
+import Exec from '../assets/exec.jpeg';
+```
+
+### Removing
+
+Open `execList.js`, find the individual and either comment them out or delete their section. Also delete their image in
+`src/pages/about/assets`.
+
+## Policy documents
```

Comment:
Where did these policy docs come from? (google drive), we should mention that so the source does not get forgotten about

Reply:
Good call